tox.ini

```diff
@@ -1,25 +1,25 @@
 [tox]
-envlist = clean,py{37,38,39,310},report
+envlist = clean,py{37,38,39,310,311},report
 
 [testenv]
 deps = -r{toxinidir}/requirements/test.txt
 parallel_show_output = true
+usedevelop = true
 depends =
-    py{37,38,39,310}: clean
-    report: py{37,38,39,310}
-# TODO: set -n auto (parallel testing) once intermittent coverage bug is fixed
-#    https://github.com/nedbat/coveragepy/issues/1514
+    py{37,38,39,310,311}: clean
+    report: py{37,38,39,310,311}
 commands =
     python --version
-    pytest -vv --cov=taskgraph --cov=src/taskgraph/run-task --cov-append --cov-report= {posargs}
+    coverage run -p --branch --source=src/taskgraph -m pytest -n auto -vv {posargs}
 
 [testenv:report]
 deps = -r{toxinidir}/requirements/test.txt
 skip_install = true
 passenv = COVERAGE_REPORT_COMMAND
 parallel_show_output = true
 commands =
-    {env:COVERAGE_REPORT_COMMAND:coverage report}
```

Comment:
This env was used in `taskcluster/ci/tests/kind.yml` to set the save location to the artifacts dir. You'll need to re-implement it so that coverage artifact is in the right place when running via CI.

Reply:
Ahhh, I never intended to fully remove it.